capstone/src/test/java/com/google/sps/servlets/student/StudentServletTest.java

```diff
@@ -0,0 +1,177 @@
+package com.google.sps.servlets;
+
+import static com.google.common.collect.ImmutableList.toImmutableList;
+import static org.mockito.Mockito.when;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.Streams;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+@RunWith(JUnit4.class)
+public final class StudentServletTest {
+  private static final String STUDENT = "student";
+  public static final String MEGHA_EMAIL = "kakm@google.com";
+  public static final String KEVIN_EMAIL = "kshao@google.com";
+  public static final String MEGAN_EMAIL = "meganshi@google.com";
+  public static final String MEGAN_NAME = "Megan Shi";
+  public static final String MEGHA_NAME = "Megha Kak";
+  public static final int YEAR_2022 = 2022;
+  public static final String MAJOR = "Computer Science";
+  public static final String CLUB_1 = "Club 1";
+
+  @Mock private HttpServletRequest request;
+  @Mock private HttpServletResponse response;
+  private StudentServlet studentServlet = new StudentServlet();
+  private LocalServiceTestHelper localHelper =
+      new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+  private DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+  private Entity student_megha;
+  private Entity student_megan;
```

Comment:
nit: camelCase?

Reply:
Oops, just changed all the variables to camel case!